src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPProjectionProvider.cs

```diff
@@ -42,6 +69,8 @@ public override async Task<ProjectionResult> GetProjectionAsync(LSPDocumentSnaps
                 throw new ArgumentNullException(nameof(position));
             }
 
+            await InitializeLogHubLoggerAsync().ConfigureAwait(false);
```

Comment:
Hmm, ditto q about why we need to ensure initialization here

Reply:
This one is a bit odder than the Diagnostics Interceptor, as I'd expect the projection provider to initialize *after* the language server. However, this is initialized prior to the Language Server, hence we need to manually initialize the logger here to ensure it's available for use. If there's a good "entry-point" where I can do this async initialization, please do let me know. I'm definitely not a fan of "scattering" the initialization in a few separate places.

Reply:
Hmm, so I'd imagine it being created before the language server (service construction) but I wouldn't imagine it being used prior to language server initialization but you're seeing that it is?

Reply:
> so I'd imagine it being created before the language server (service construction) but I wouldn't imagine it being used prior to language server initialization

That's an accurate description of what's happening. Given that behavior is expected, updated the logic so we only do logger provider initialization here (instead of also initializing the log hub itself).